cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -1,18 +1,22 @@
 apiVersion: v1beta3
 kind: ReplicationController
 metadata: 
+  name: monitoring-heapster-v1
+  namespace: default
   labels: 
-    name: heapster
+    k8s-app: heapster
+    version: v1
     kubernetes.io/cluster-service: "true"
```

Comment:
Actually I found that label values must be strings, so this has to stay as it is.